pkg/deploy/lattice/target_group_synthesizer_test.go

```diff
@@ -306,6 +283,66 @@ func Test_SynthersizeTriggeredByServiceImport(t *testing.T) {
 	}
 }
 
+func Test_SynthesizeTriggeredTargetGroupsDeletion_TriggeredByServiceImport(t *testing.T) {
```

Comment:
ServiceImport should `never` trigger deletion of target group!  Target group should be created or deleted by "ServiceExport".